lib/backend/k8s/resources/workloadendpoint.go

```diff
@@ -126,6 +126,12 @@ func (c *WorkloadEndpointClient) Get(ctx context.Context, key model.Key, revisio
 	if err != nil {
 		return nil, err
 	}
+	if wepID.Pod == "" {
```

Comment:
Surely this needs to be `wepIP.Pod == "" && wepID.Workload == ""`? For non-k8s the pod will be `""`

Reply:
oops, nvm. This is in the k8s backend.